### PR TITLE
Gender-Neutral

### PR DESCRIPTION
**Pull request by MrUnknownGamer**
Added gender-neutral stuff for the only things I could find.

Also did Spanish (Anyone else can do other languages, that would be
great!)

P.S How did someone forget to put the "ser" conjugation on the "He was
an innocent" it basically said "He was innocent" in Spanish :P Fixed
that too.

### CODE REVIEW

**Comment by Bo98** [comment]
What's wrong with saying "They were innocent"? Sounds better than "They were an innocent".

**Comment by iSnipeu** [comment]
I'm getting deja vu here, has this not happened before? Also I agree with it Bo98, it sounds better without the "an".

**Comment by Kefta** [comment]
Era is already gender neutral. Estaba is not the direct translation of "they," rather, it refers to plurality

**Comment by Kefta** [comment]
And I did not put an "un" before innocent because innocent in Spanish is not a noun, it's an adjective. Just like English

**Comment by svdm** [comment]
Can't speak for the Spanish, but the English should indeed be "They were innocent" or "They were an innocent Terrorist".

**Comment by MrUnknownGamer** [comment]
Ok, ill fix it then.

**Comment by KateAdams** [comment]
:+1:

**Comment by Kefta** [comment]
You didn't fix?

**Comment by MrUnknownGamer** [comment]
Opps! Sorry.

**Comment by Kefta** [comment]
Estaban still isn't correct.

**Comment by MrUnknownGamer** [comment]
How so?

**Comment by Kefta** [comment]
Estaban means multiple people; it's not a direct translation of they. Era is correct.

**Comment by KateAdams** [comment]
@Kefta 'They' is also plural in English, but is acceptable for singular use for gender-neutral speech.  Is 'era' gender neutral *and* singular?

**Comment by Kefta** [comment]
@KateAdams Without any subject (el, ella, usted), Era is gender neutral and singular. Google translate comes back as "he" because there is no gender-neutral singular in English that can't also be a plural other than "it," which is not appropriate for people. Estaban, although gender neutral, is not singular in any way.

**Comment by LennyPenny** [comment]
It is appropiate for people if you want to be gender neutral

**Comment by Kefta** [comment]
@svdm Ready

File: garrysmod/gamemodes/terrortown/gamemode/lang/english.lua [hunk 1]
```diff
@@ -792,7 +792,7 @@ L.col_total  = "Total points"
 L.something      = "something"
 
 -- Kill events
-L.ev_blowup      = "{victim} blew himself up"
+L.ev_blowup      = "{victim} blew themself up"
```
Comment (zerfgog) [inline garrysmod/gamemodes/terrortown/gamemode/lang/english.lua]: shouldn't this be themselves?

Comment (KateAdams) [inline garrysmod/gamemodes/terrortown/gamemode/lang/english.lua]: @zerfgog No, themself is explicitly singular.

Comment (robotboy655) [inline garrysmod/gamemodes/terrortown/gamemode/lang/english.lua]: http://blog.oxforddictionaries.com/2013/01/themself/

**Pull request merged**